empyrical/stats.py

```diff
@@ -152,6 +152,40 @@ def cum_returns(returns, starting_value=0):
         return df_cum * starting_value
 
 
+def total_returns(returns, starting_value=0):
+    """
+    Compute total returns from simple returns.
+
+    Parameters
+    ----------
+    returns : pd.Series or np.ndarray
+        Returns of the strategy as a percentage, noncumulative.
+         - Time series with decimal returns.
+         - Example:
+            2015-07-16    -0.012143
+            2015-07-17    0.045350
+            2015-07-20    0.030957
+            2015-07-21    0.004902.
+    starting_value : float, optional
+       The starting returns.
+
+    Returns
+    -------
+    float
+
+    """
+
+    if len(returns) == 0:
+        return np.nan
+
+    cumulative_returns = cum_returns(returns, starting_value)
+
+    if isinstance(cumulative_returns, np.ndarray):
+        return cumulative_returns[-1]
+    else:
+        return cum_returns(returns, starting_value).iloc[-1]
```

Comment:
Looks like in this case we calculate the `cum_returns` twice!

I think, instead of doing an instance check in order to know the return type of `cum_returns` so we can get the last item, it's likely much faster to always pull the array out of the series and pass that through.  Then you'll also know how to index into the returned array.  (See https://github.com/quantopian/empyrical/blob/1dc28d51b16a55350c09ca60d1b3c2ed4be78ee5/empyrical/stats.py#L334-L336)